Sources/SwiftUICharts/Charts/LineChart/MarkerShape.swift

```diff
@@ -0,0 +1,25 @@
+import SwiftUI
+
+struct MarkerShape: Shape {
+    var data: [(Double, Double)]
+    func path(in rect: CGRect) -> Path {
+        let path = Path.drawChartMarkers(data: data, in: rect)
+        return path
+    }
+}
+
+struct MarkerShape_Previews: PreviewProvider {
+    static var previews: some View {
+        Group {
+            MarkerShape(data: [(0, 0), (0.25, 0.5), (0.5,0.8), (0.75, 0.6), (1, 1)])
+                .stroke()
+                .rotationEffect(.degrees(180), anchor: .center)
```

Comment:
I think I would put it into `View` extension as it's keep being used on many places. Maybe something like this?

```
extension View {
    func toStandardCoordinateSystem() -> some View {
        self
            .rotationEffect(.degrees(180), anchor: .center)
            .rotation3DEffect(.degrees(180), axis: (x: 0, y: 1, z: 0))
    }
}
```

Reply:
done :)